### ISSUE TITLE
support for Java v9 GC log formatting?

### ISSUE THREAD

**Issue by eostermueller** [dmart28/gcplot#13]
Hi, do you have plans to support the new GC log formatting with JDK 9 and later?
...detailed here:
https://dzone.com/articles/disruptive-changes-to-gc-logging-in-java-9

Thanks for sharing this interesting utility.

--Erik

**Comment by dmart28** [comment]
@eostermueller no, for now there are no such plans, since I've lack of time to support gcplot properly.

**Comment by eladtamary** [comment]
Hi @dmart28, 
I used your project to investigate OOM issues we had and it was great 👍 .
I loved the simplicity and ease-of-use of it.

Knowing that you lack the time to support this project, are you open to pull requests?

Thanks,
Elad
